pallets/xc-asset-config/src/weights.rs

```diff
@@ -55,147 +60,65 @@ pub trait WeightInfo {
-	}
-	// Storage: XcAssetConfig AssetIdToLocation (r:1 w:1)
-	// Proof Skipped: XcAssetConfig AssetIdToLocation (max_values: None, max_size: None, mode: Measured)
-	// Storage: EVM AccountCodes (r:0 w:1)
-	// Proof Skipped: EVM AccountCodes (max_values: None, max_size: None, mode: Measured)
-	// Storage: XcAssetConfig AssetLocationUnitsPerSecond (r:0 w:1)
-	// Proof Skipped: XcAssetConfig AssetLocationUnitsPerSecond (max_values: None, max_size: None, mode: Measured)
-	// Storage: XcAssetConfig AssetLocationToId (r:0 w:1)
-	// Proof Skipped: XcAssetConfig AssetLocationToId (max_values: None, max_size: None, mode: Measured)
-	fn remove_asset() -> Weight {
-		// Minimum execution time: 18_645 nanoseconds.
-		Weight::from_parts(18_878_000, 0)
-			.saturating_add(Weight::from_parts(0, 2987))
-			.saturating_add(T::DbWeight::get().reads(1_u64))
-			.saturating_add(T::DbWeight::get().writes(4_u64))
-	}
-
-	fn update_migration_step() -> Weight {
-		// Minimum execution time: 18_645 nanoseconds.
-		Weight::from_parts(18_878_000, 0)
-			.saturating_add(Weight::from_parts(0, 2987))
-			.saturating_add(RocksDbWeight::get().reads(1_u64))
-			.saturating_add(RocksDbWeight::get().writes(4_u64))
-	}
-}
-
-// For backwards compatibility and tests
-impl WeightInfo for () {
-	// Storage: XcAssetConfig AssetIdToLocation (r:1 w:1)
-	// Proof Skipped: XcAssetConfig AssetIdToLocation (max_values: None, max_size: None, mode: Measured)
-	// Storage: EVM AccountCodes (r:0 w:1)
-	// Proof Skipped: EVM AccountCodes (max_values: None, max_size: None, mode: Measured)
-	// Storage: XcAssetConfig AssetLocationToId (r:0 w:1)
-	// Proof Skipped: XcAssetConfig AssetLocationToId (max_values: None, max_size: None, mode: Measured)
-	fn register_asset_location() -> Weight {
-		// Minimum execution time: 15_540 nanoseconds.
-		Weight::from_parts(16_114_000, 0)
-			.saturating_add(Weight::from_parts(0, 2493))
-			.saturating_add(RocksDbWeight::get().reads(1_u64))
-			.saturating_add(RocksDbWeight::get().writes(3_u64))
+			.saturating_add(T::DbWeight::get().reads(1))
+			.saturating_add(T::DbWeight::get().writes(2))
 	}
-	// Storage: XcAssetConfig AssetLocationToId (r:1 w:0)
-	// Proof Skipped: XcAssetConfig AssetLocationToId (max_values: None, max_size: None, mode: Measured)
-	// Storage: XcAssetConfig AssetLocationUnitsPerSecond (r:0 w:1)
-	// Proof Skipped: XcAssetConfig AssetLocationUnitsPerSecond (max_values: None, max_size: None, mode: Measured)
 	fn set_asset_units_per_second() -> Weight {
-		// Minimum execution time: 15_297 nanoseconds.
-		Weight::from_parts(15_551_000, 0)
-			.saturating_add(Weight::from_parts(0, 2661))
-			.saturating_add(RocksDbWeight::get().reads(1_u64))
-			.saturating_add(RocksDbWeight::get().writes(1_u64))
+		// Proof Size summary in bytes:
+		//  Measured:  `93`
+		//  Estimated: `0`
+		// Minimum execution time: 13_646_000 picoseconds.
+		Weight::from_parts(13_783_000, 0)
+			.saturating_add(Weight::from_parts(0, 0))
```

Comment:
The proof size is 0 now with frame-omni-bencher v1, I am not sure if this is ok. I think it's because in the `--json-input` they are empty.